nix/modules/tezos-endorser.nix

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+#
+# SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+  tezos-endorser-pkgs = {
+    "007-PsDELPH1" =
+      "${pkgs.ocamlPackages.tezos-endorser-007-PsDELPH1}/bin/tezos-endorser-007-PsDELPH1";
+    "008-PtEdo2Zk" =
+      "${pkgs.ocamlPackages.tezos-endorser-008-PtEdo2Zk}/bin/tezos-endorser-008-PtEdo2Zk";
+    "009-PsFLoren" =
+      "${pkgs.ocamlPackages.tezos-baker-009-PsFLoren}/bin/tezos-baker-009-PsFLoren";
+  };
+  common = import ./common.nix { inherit lib; };
+  cfg = config.services.tezos-endorser;
+  instanceOptions = types.submodule ( {...} : {
+    options = common.daemonOptions // {
```

Comment:
`tezos-endorser` and `tezos-baker` also should have an option for setting up baker/endorser account alias and use this alias in `tezos-endorser ... run ${cfg.bakerAccountAlias}` or `tezos-baker ... run with local node "$STATE_DIRECTORY/node/data"  ${cfg.endorserAccountAlias}`